frontend/include/chpl/resolution/resolution-queries.h

```diff
@@ -314,6 +314,15 @@ types::QualifiedType returnType(ResolutionContext* rc,
                                 const TypedFnSignature* sig,
                                 const PoiScope* poiScope);
```

Comment:
Could you update the docs comment for `returnType` to describe what happens for iterators (which is, it returns an `IteratorType`, right?)